src/lib/plugin/index.spec.ts

```diff
@@ -0,0 +1,92 @@
+import * as Path from 'path'
+import * as TC from '../test-context'
+import { repalceInObject } from '../utils'
+import { getPluginManifest } from './manifest'
+import { Plugin } from './types'
+
+const ctx = TC.create(TC.tmpDir(), TC.fs())
+
+describe('manifest', () => {
+  let plugin: Plugin
+  const run = () => repalceInObject(ctx.tmpDir, '<project root>', getPluginManifest(plugin))
```

Comment:
I know it's out of scope for the PR, but we really need to rename that function, it burns my eyes every time I see it 😂